pyproject.toml

```diff
@@ -24,3 +24,10 @@ exclude = '''
   | .gitignore
 )
 '''
+
+[tool.isort]
+profile = "black"
+line_length = 120
+
+[tool.pyright]
+include = ["neuralprophet/configure.py"]
```

Comment:
That looks like the wrong project, might make sense to start with proper typing for all files right from the start